docs/conf.py

```diff
@@ -0,0 +1,332 @@
+
+# Preview image URL
+#
+# TODO: To customise the preview image, update as needed.
+
+ogp_image = "https://assets.ubuntu.com/v1/253da317-image-document-ubuntudocs.svg"
+
+
+# Product favicon; shown in bookmarks, browser tabs, etc.
+
+# TODO: To customise the favicon, uncomment and update as needed.
+
+# html_favicon = '.sphinx/_static/favicon.png'
+
+
+# Dictionary of values to pass into the Sphinx context for all pages:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_context
+
+html_context = {
+    # Product page URL; can be different from product docs URL
+    #
+    # TODO: Change to your product website URL,
+    #       dropping the 'https://' prefix, e.g. 'ubuntu.com/lxd'.
+    #
+    # TODO: If there's no such website,
+    #       remove the {{ product_page }} link from the page header template
+    #       (usually .sphinx/_templates/header.html; also, see README.rst).
+    "product_page": "documentation.ubuntu.com",
+    # Product tag image; the orange part of your logo, shown in the page header
+    #
+    # TODO: To add a tag image, uncomment and update as needed.
+    # 'product_tag': '_static/tag.png',
+    # Your Discourse instance URL
+    #
+    # TODO: Change to your Discourse instance URL or leave empty.
+    #
+    # NOTE: If set, adding ':discourse: 123' to an .rst file
+    #       will add a link to Discourse topic 123 at the bottom of the page.
+    "discourse": "https://discourse.ubuntu.com",
+    # Your Mattermost channel URL
+    #
+    # TODO: Change to your Mattermost channel URL or leave empty.
+    "mattermost": "https://chat.canonical.com/canonical/channels/documentation",
+    # Your Matrix channel URL
+    #
+    # TODO: Change to your Matrix channel URL or leave empty.
+    "matrix": "https://matrix.to/#/#documentation:ubuntu.com",
+    # Your documentation GitHub repository URL
+    #
+    # TODO: Change to your documentation GitHub repository URL or leave empty.
+    #
+    # NOTE: If set, links for viewing the documentation source files
+    #       and creating GitHub issues are added at the bottom of each page.
+    "github_url": "https://github.com/canonical/multipass",
+    # NOTE TO GIULIA: This line was suggested by Shane to enable the feedback button,
+    # but it doesn't work at the moment.
+    # 'github_issues': 'enabled',
+
+    # Docs branch in the repo; used in links for viewing the source files
+    #
+    # TODO: To customise the branch, uncomment and update as needed.
+    # 'github_version': 'main',
+    # Docs location in the repo; used in links for viewing the source files
+    #
+    # TODO: To customise the directory, uncomment and update as needed.
+    "github_folder": "/docs/",
+    # TODO: To enable or disable the Previous / Next buttons at the bottom of pages
+    # Valid options: none, prev, next, both
+    # "sequential_nav": "both",
+    # TODO: To enable listing contributors on individual pages, set to True
+    "display_contributors": False,
+}
```

Comment:
```suggestion
    "github_issues": "enabled",
}
```

In my local testing, adding this line enabled the "Give feedback" button.
If this is something you want, test it on your machine to check.

You could then remove the notes about the feedback button elsewhere in the `html_context` scope.

Reply:
Done! Thanks, it works for me as well.